features/smoke_tests/handled.feature

```diff
@@ -30,6 +30,7 @@ Scenario: Notify caught Java exception with default configuration
     And the error payload field "events.0.threads.0.stacktrace.0.method" ends with "getThreadStackTrace"
 
     # App data
+    And the event "app.binaryArch" is not null
```

Comment:
Not unless these tests are guaranteed to always run on the same architecture, which I wasn't sure was the case.

Reply:
I was thinking that it'd make sense to validate that it was just _any_ of one of the available architectures. This might sound like overkill but the test gap coverage has bitten us in the past: https://github.com/bugsnag/bugsnag-android/pull/1220

Reply:
Ah yeah, validate that its in the list makes sense.